roles/cloud_connector/templates/rhc-worker-cloud-connector.toml.j2

```diff
@@ -0,0 +1,5 @@
+env = [
+  "FORWARDER_USER={{ foreman_cloud_connector_username }}",
+  "FORWARDER_PASSWORD={{ foreman_cloud_connector_password }}",
+  "FORWARDER_URL={{ foreman_cloud_connector_url }}"
```

Comment:
```suggestion
  "FORWARDER_URL={{ foreman_cloud_connector_url }}",
  "FORWARDER_HANDLER=foreman_rh_cloud"  
```
We need to specify the handler in case we're not using the batch wrapper (newer yggdrasil)

Reply:
Got a reference? Right now we derive the handler from the file name, you want to just set it it explicitly?

Reply:
We are setting it in the code only if `CONFIG_FILE` is set here: https://github.com/ShimShtein/yggdrasil-worker-forwarder/blob/main/main.go#L36, but if the variable is empty (which would be the case for newer yggd version) we are expecting it to be set explicitly [here](https://github.com/ShimShtein/yggdrasil-worker-forwarder/blob/main/main.go#L47).

Yggdrasil [does something similar](https://github.com/RedHatInsights/yggdrasil/blob/b35da0f6e66c6fbac58ec9839925bafdddf28b0b/cmd/yggd/worker.go#L42), but it uses this for different purposes (like naming the pid file), and does not reuse the information from the worker (the one that we send from the `FORWARDER_HELPER` variable)